docs/web-apps/automated-testing/selenium/quickstart.md

```diff
@@ -0,0 +1,217 @@
+
+
+### Step 3: View Test Results on Sauce Labs
+The expected response is a session confirmation message and hyperlink that will take you to your test in progress and [results](/test-results/viewing-test-results/) on Sauce Labs.<br/><img src={useBaseUrl('img/selenium/basic-testresults.png')} alt="basic-testresults" />
+
+
+## Selenium WebDriver Test
+
+In this Selenium test, you'll automate WebDriver to open and close a Chrome browser window.
+
+### Step 1: Clone Project
+From your IDE's terminal, clone the [Sauce Labs Selenium Training](https://github.com/saucelabs-training/advanced-selenium) repository to your home directory.
+  ```bash
+  git clone https://github.com/saucelabs-training/advanced-selenium.git
+  ```
+
+### Step 2: Download Dependencies
+This quickstart test utilizes the Java Development Kit (JDK) ([download here](https://www.oracle.com/java/technologies/downloads/#java8)), which is required to compile and execute Java code, and Selenium Java bindings ([download here](https://www.selenium.dev/downloads/)), which are required for Selenium-Java compatibility.
+
+
+### Step 3: Import Dependencies
+
+1. From your IDE, open the **advanced-selenium** project. The test file is located under **src/test/java/com/saucelabs/advancedselenium/SeleniumTest.java**. <br/><img src={useBaseUrl('img/selenium/selenium-test.png')} alt="selenium-test" />
+2. If it's your first time ever opening a Java project, your IDE will prompt you to set up your JDK. You'll need to:
+   * Click on the prompt.<br/><img src={useBaseUrl('img/selenium/jdk-setup.png')} alt="jdk-setup" width="400"/><br/>
+   * Provide the location where you downloaded JDK. Some IDEs will detect and find the path for you, so that all you have to do is select it.<br/><img src={useBaseUrl('img/selenium/jdk-config.png')} alt="jdk-config" width="500"/>
+
+:::tip
+If you're new to IDEs, here are some tips:
+<details><summary>OpenJDK distributions</summary>
+IDEs also give you the option to download open source JDK distributions (e.g., Amazon Corretto, Oracle OpenJDK, and Eclipse Temurin) as an alternative to downloading JDK to your local machine. This works, but will only download it this specific project. If you're setting up a permanent developer environment that will be used beyond this Quickstart, you should download the JDK locally.<br/><img src={useBaseUrl('img/selenium/jdk-config-alt.png')} alt="jdk-config" width="500"/>
+</details>
+
+<details><summary>Extensions and plugins</summary>
+If your IDE recommends additional extensions and plugins for Java and Maven, such as the ones listed below, you should accept them since they may contain more dependencies needed to run a test.<br/><img src={useBaseUrl('img/selenium/java-webdriver-dependencies.png')} alt="java-webdriver-dependencies" width="350"/>
+</details>
+:::
+
+### Step 4: Run Test
+Right-click on the **SeleniumTest.java** test file and run it.<br/><img src={useBaseUrl('img/selenium/run-selenium.png')} alt="run-selenium" />
```

Comment:
Hi @titusfortner @diemol , could you please take a look at this Selenium Quickstart doc? 

I've run into an issue with this part (lines 28-88). This is the version of the `advanced-selenium` repo I used: https://github.com/saucelabs-training/advanced-selenium/tree/v2.1. With the recent changes, I no longer see `SeleniumTest.java`. Is it possible for customers to clone that older 2.1 version of the repo?